src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
The class `IsoscelesTrapezoid` should not start with an empty line. Please remove the empty line before the class declaration.